status/status.go

```diff
@@ -55,14 +58,12 @@ func NewAdapter(logger logr.Logger, k8sClient client.Client, opts ...AdapterOpti
 	return &adapter
 }
 
-// GetReporters returns a list of enabled/supported status reporters for a PipelineRun.
+// GetReporters returns enabled/supported status reporters for a PipelineRun or a Snapshot.
 // All potential reporters must be added to this function for them to be utilized.
-func (a *Adapter) GetReporters(pipelineRun *tektonv1beta1.PipelineRun) ([]Reporter, error) {
-	var reporters []Reporter
-
-	if helpers.HasLabelWithValue(pipelineRun, gitops.PipelineAsCodeGitProviderLabel, gitops.PipelineAsCodeGitHubProviderType) {
-		reporters = append(reporters, a.githubReporter)
+func (a *Adapter) GetReporter(object client.Object) Reporter {
```

Comment:
We have only one reporter for one object, so I feel we can remote the  `[]` usage here.

Reply:
I think that the original intent was to support multiple reporters for the future.
If it's not too large of an obstacle, it might make sense to keep the current structure and keep the possibility for a future support for other cms providers.

Reply:
So let me change it back. Thanks.